src/js_native_api_types.h

```diff
@@ -115,4 +115,25 @@ typedef struct {
   napi_status error_code;
 } napi_extended_error_info;
 
+#ifdef NAPI_EXPERIMENTAL
+typedef enum {
+  napi_key_include_all_properties,
```

Comment:
```suggestion
  napi_key_include_prototypes,
```

This doesn't seem to be aligned with docs.